### ISSUE TITLE
poly diff output wraps at 80 chars

### ISSUE THREAD

**Issue by hstravis** [DavidVujic/python-polylith#245]
The output from the `poetry poly diff --bricks --short` command wraps at 80 characters. This is unexpected since the output is intended to be used programmatically.

Work-around in bash:

```bash
poetry poly diff --bricks --short --since stable | tr -d \\r | tr -d \\n
```

**To Reproduce**
1. Open a terminal window 
2. Go to root of the project
4. Run the command `poetry poly diff --bricks --short --since stable`
5. If there are more than 80 chars of output, the line will wrap with `\r\n`

**Error Log**
If applicable, add the error logs.

**Expected behavior**
No new lines in output of poly diff command with `--bricks --short`.

**Screenshots**
Sorry, it's a private repo.

**Desktop (please complete the following information):**
 - OS: Windows
 - Python version: 3.11.7
 - Poetry version: 1.8.3

**Additional context**
Polylith plugin version: 1.26.2

**Comment by DavidVujic** [comment]
Thank you for reporting this issue @hstravis - I'll have a look right away.

**Comment by DavidVujic** [comment]
I have published a new version (1.27.1 of the Poetry plugin) that I believe will fix this issue. Thank you again for reporting ⭐
